Configure.pl

```diff
@@ -53,10 +53,15 @@
 
     unless (defined $options{prefix}) {
         my $default = defined($options{sysroot}) ? '/usr' : File::Spec->catdir(getcwd, 'install');
-        print "ATTENTION: no --prefix supplied, building and installing to $default\n";
+        # print "ATTENTION: no --prefix supplied, building and installing to $default\n";
         $options{prefix} = $default;
     }
     $options{prefix} = File::Spec->rel2abs($options{prefix});
+    unless (defined $options{libdir}) {
+        my $default = File::Spec->catdir($options{prefix}, 'lib');
+        # print "ATTENTION: no --libdir supplied, building and installing architecture specific files to $default\n";
```

Comment:
Committing commented out code is rarely useful. It's either meant to be run or not.